self-paced-labs/image_build/kaniko_lab/scripts/pod.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kaniko
+spec:
+  containers:
+  - name: kaniko
+    image: gcr.io/kaniko-project/executor:latest
+    args: ["--dockerfile=/workspace/dockerfile",
+            "--context=dir://workspace",
+            "--destination=<user-name>/<repo-name>"] # replace with your dockerhub account
```

Comment:
Suggest to use Upper Case for highlight variable, like <USER-NAME>/<REPO-NAME> instead of <user-name>/<repo-name>